### PR TITLE
Removed useless code

### PR DESCRIPTION
**Pull request by lupakivan**
#### Please tick a box ###
- [x] **refactor** _(A code change that neither fixes a bug nor adds a feature_)

#### If you're adding a feature, is it documented in a storybook story?
- [ ] Yes
- [ ] No

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/atoms/Button.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const Button = styled.button`
 	position: relative;
 	text-decoration: none;
 	text-align: center;
-	border-radius: 0;
+	border-radius: ${props => props.borderRadius || 0}%;
```
Comment (tnlitv) [inline src/atoms/Button.js]: maybe use defaultProps? :)

File: src/atoms/MainRecipe/IngredientsSection.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const Counter = Styled.span`
 		color: #949494;
 		font-size: 2.4rem;
 		margin: .3rem .6rem;
-    display: inline-block;
+    	display: inline-block;
```
Comment (sergeyvlakh) [inline src/atoms/MainRecipe/IngredientsSection.js]: Indent

File: src/atoms/Button.js (hunk 1 above)
Comment (misund) [reply src/atoms/Button.js]: +1

Comment (misund) [reply src/atoms/Button.js]: Include the percentage sign in the prop value as well.

File: src/atoms/Button.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ const Button = styled.button`
 	position: relative;
 	text-decoration: none;
 	text-align: center;
-	border-radius: 0;
+	border-radius: ${props => `${props.borderRadius}%`};
```
Comment (misund) [inline src/atoms/Button.js]: Please send the percentage sign as part of the prop.

Comment (misund) [reply src/atoms/Button.js]: Like this
```
${props => `${props.borderRadius}`};
```

**Pull request merged**